test/integration/import_test.go

```diff
@@ -0,0 +1,34 @@
+//go:build integration
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/checkmarx/ast-cli/internal/params"
+	"gotest.tools/assert"
+)
+
+func TestImport_ImportSarifFileWithCorrectFlags_CreateImportSuccessfully(t *testing.T) {
+	projectId, projectName := createProject(t, nil, nil)
+	defer deleteProject(t, projectId)
+	args := []string{
+		"import",
+		flag(params.ProjectName), projectName,
+		flag(params.ImportFilePath), "./data/sarif.sarif",
+	}
+	err, _ := executeCommand(t, args...)
+	assert.NilError(t, err, "import failed")
+}
+
+func TestImport_ImportSarifFileProjectDoesntExist_CreateImportWithProvidedNewNameSuccessfully(t *testing.T) {
+	projectName := projectNameRandom
+
+	args := []string{
+		"import",
+		flag(params.ProjectName), projectName,
+		flag(params.ImportFilePath), "./data/sarif.sarif",
+	}
+	err, _ := executeCommand(t, args...)
+	assert.NilError(t, err, "import failed")
+}
```

Comment:
Note: on a separate task, we should make it possible to delete a project by name

Reply:
why on different? we already had a general cleanup test, let's have a cleanup here as well

Reply:
you can do it by get and delete

Reply:
BTW, if the project doesn't exist we agreed on creating new one. so why did this test assert fail?

Reply:
Done